pkg/virt-handler/container-disk/mount.go

```diff
@@ -152,25 +146,13 @@ func (m *mounter) getMountTargetRecord(vmi *v1.VirtualMachineInstance) (*vmiMoun
 	}
 
 	// if not there, see if record is on disk, this can happen if virt-handler restarts
-	recordFile := filepath.Join(m.mountStateDir, filepath.Clean(string(vmi.UID)))
```

Comment:
Why did we need filepath.Clean here?

Reply:
I believe this was just in case handling. You can see that nowhere else we do this, in other words we diverge.

See f68180d781d0cd96ef18e3caa3f54e8456faeb37 (gosec era)